### ISSUE TITLE
splitPages not working?

### ISSUE THREAD

**Issue by AshotN** [nisaacson/pdf-text-extract#18]
I'm trying to merge all the PDF pages together but splitPages: false doesn't seem to be doing anything for me

Here is my code (its in CoffeeScript)

```
extract pdfFilePath, { splitPages: false }, (err, text) ->
    if err
        return console.dir err

    console.log text
    res.end text
```

**Comment by zettacristiano** [comment]
+1

**Comment by webkonglong** [comment]
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn pdftotext ENOENT
    at exports._errnoException (util.js:1018:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at Module.runMain (module.js:606:11)
    at run (bootstrap_node.js:390:7)
    at startup (bootstrap_node.js:150:9)
    at bootstrap_node.js:505:3

**Comment by jyc913** [comment]
splitpages is still not working. I downloaded from npm, and the version is 1.5.0

**Comment by AshotN** [comment]
Sadly I don't think this project is seeing much updates.

**Comment by aready87** [comment]
I believe none of the options are working.
I tested firstPage, lastPage, splitPages and crop. None of them changes the extract output.

**Comment by aready87** [comment]
Has anyone been able to get the options to work?

**Comment by aready87** [comment]
Ok, I figured what the issue is.
I believe we all pulled the repo via npm or repo. The npm registry for this project is 3 years old.
The fix was introduced in Pull #27 which was in January 2019. 398a5371123211d84f402eb6809db07752571337

So you have two options, make the edit to the library yourself or clone using the latest repo using the github url.

With the fix, all the options are working.

**Comment by AshotN** [comment]
You can add a package to package.json with its github URL as well
